### ISSUE TITLE
Mouse Tracking: NVDA does not read computed name for aria-label

### ISSUE THREAD

**Issue by aminimalanimal** [nvaccess/nvda#5600]
NVDA does not read content provided within `aria-label` when hovering over items with mouse tracking on. Instead, only visible content can be heard.

[Demonstration on Codepen](http://s.codepen.io/aminimalanimal/debug/xZGmVr)

**Comment by leonardder** [comment]
This is expected behaviour. The aria label for an element is not visible on screen, and for mouse tracking, only visible information is announced. Closing as invalid.
